packages/asc-ui/src/components/ErrorMessage/ErrorMessage.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+import { ThemeProvider } from '../../theme'
+import ErrorMessage from './ErrorMessage'
+
+describe('ErrorMessage', () => {
+  it('should render correctly', () => {
+    const { queryByText, queryByTestId } = render(
+      <ThemeProvider>
+        <ErrorMessage message="Dit veld is verplicht" />
```

Comment:
Nitpicking: this package is using only English text throughout the app

Reply:
gedaan